src/openms/source/FORMAT/DATAACCESS/SiriusFragmentAnnotation.cpp

```diff
@@ -266,7 +266,7 @@ namespace OpenMS
         candidates.getRow(i, sl);
         String adduct = sl[columnname_to_columnindex.at("adduct")];
         adduct.erase(std::remove_if(adduct.begin(), adduct.end(), ::isspace), adduct.end());
-        rank_filename.emplace(std::make_pair(sl[columnname_to_columnindex.at("rank")].toInt(),
+        rank_filename.emplace(std::make_pair(sl[columnname_to_columnindex.at("formulaRank")].toInt(),
```

Comment:
did this change?

Reply:
Yes, with the latest SIRIUS the column is named formulaRank.